modules/doc/content/help/inl/index.md

```diff
@@ -10,7 +10,7 @@ available codes and how to utilize the [!ac](INL) resources to run applications.
 - [inl/hpc_ondemand.md]
 
 
-## Requesting Access id=access
+# Requesting Access id=access
```

Comment:
Git rid of the right table of contents bar. We don't need it for one item...

Reply:
👍🏻